src/core/sprites/webgl/SpriteRenderer.js

```diff
@@ -251,10 +253,12 @@ export default class SpriteRenderer extends ObjectRenderer
 
             nextTexture = sprite._texture.baseTexture;
 
-            if (blendMode !== sprite.blendMode)
+            const spriteBlendMode = premultiplyBlendMode[Number(nextTexture.premultipliedAlpha)][sprite.blendMode];
```

Comment:
After I saw `correctBlendMode` I'm curious why this isn't using that function but is instead duplicating the same code?